frontend/src/Stats.vue

```diff
@@ -116,13 +135,21 @@ export default {
         return null;
       }
 
+      const labels = history.flatMap((point) => point.date);
+      const data = history.flatMap((point) => point.total);
+
+      console.log(labels, data);
+
       return {
-        series: [
+        labels: labels,
+        datasets: [
           {
-            name: "Total issues",
-            data: history.map((point) => {
-              return { x: new Date(point.date), y: point.total };
-            }),
+            borderColor: "rgb(75, 192, 192)",
+            pointBorderColor: "rgb(75, 192, 192)",
```

Comment:
I used dark red but maybe you can just give me the hex code for the actual color, I don't know if maybe the URL I found in the docs is not good? But I simply cannot see this page so I have no idea what the graph should look like

![Screenshot from 2024-10-09 18-18-47](https://github.com/user-attachments/assets/b2018157-1d45-4d9d-bda5-d19e910ee4b4)

Reply:
This cannot be tested at the moment because the production API endpoints queried
https://api.code-review.moz.tools/v1/check/stats/?since=2024-09-10
https://api.code-review.moz.tools/v1/check/history/?since=2024-09-10
fail to respond at the moment after loading for some time.